### ISSUE TITLE
Container as handler for Queue or Cron

### ISSUE THREAD

**Issue by apetta** [sst/ion#951]
Hey, is there any guidance available for attaching a fargate container to Queues or Crons?

**Comment by jayair** [comment]
I don't think we have an example for this on our side. Is there a doc or tutorial you've seen elsewhere that implements this? I can take a look and see how it would work in SST.

**Comment by apetta** [comment]
> I don't think we have an example for this on our side. Is there a doc or tutorial you've seen elsewhere that implements this? I can take a look and see how it would work in SST.

Hmm, no example unfortunately. Was looking for a way to run some python as a handler for Queue/Cron. It looks like the recently added python function support might address this...

**Comment by thdxr** [comment]
for a container + queue it would have to poll SQS directly, can look up how or ask an LLM it should generate it pretty well<br><br>cron is a bit trickier since the container will need to be spun up?
